deploy/charts/emqx-enterprise/templates/StatefulSet.yaml

```diff
@@ -138,6 +141,9 @@ spec:
             subPath: "emqx.lic"
             readOnly: true
           {{- end }}
+          {{- if .Values.extraVolumeMounts }}
+            {{- toYaml .Values.extraVolumeMounts | nindent 12 }}
```

Comment:
@Rory-Z I'm testing it out locally and I think the indentation should be 10 here, to be in line with the YAML indentation for the other volume mounts